test/functional/isVisible.js

```diff
@@ -0,0 +1,58 @@
+const { assert } = require('chai');
+const firefox = require('selenium-webdriver/firefox');
+const webdriver = require('selenium-webdriver');
+const { ancestors, isDomElement, isVisible, toDomElement } = require('../../utilsForFrontend');
+
+const { until, By } = webdriver;
+
+const WAIT_MS = 10000;
+const TEST_PAGE_URL = 'http://localhost:8000/isVisible.html';
+
+describe('isVisible', () => {
+    const options = new firefox.Options();
+    options.headless();
+
+    const driver = new webdriver.Builder()
+        .forBrowser('firefox')
+        .setFirefoxOptions(options)
+        .build();
+
+    it('Should return false when an element is hidden', async () => {
```

Comment:
So far, the rest of the codebase starts `it(` operands with a lowercase letter. Those all show up in the test output, so they're pretty conspicuous.